twofactor_privacyidea/lib/Provider/TwoFactorPrivacyIDEAProvider.php

```diff
@@ -187,13 +187,18 @@ private function triggerChallenges($username)
                     if (property_exists($detail, "multi_challenge")) {
                         $multi_challenge = $detail->multi_challenge;
                         if (count($multi_challenge) === 0) {
-                        	$this->session->set("pi_hideOTPField", false);
+							$this->session->set("pi_hideOTPField", false);
 						} else {
 							for ($i = 0; $i < count($multi_challenge); $i++) {
 								if ($multi_challenge[$i]->type === "u2f") {
```

Comment:
We should start using a switch now, I think.

Reply:
Resolved in next commit.